vuestorefront/magento2
# fix(theme): fix missing address details tab for non-english locale stores

## Description
fix missing address details tab for non-english locale stores

## Related Issue
#535 

## Motivation and Context
bugfix

## How Has This Been Tested?

1. Log in/register as a customer
2. Change locale to German
3. Navigate to my account panel
4. Click Adressendetails in side menu
5. Click "Neue Adresse hinzufügen" button
6. Observe that tab's content is displayed properly

## Screenshots (if appropriate):
![Screenshot 2022-02-02 at 12 02 55](https://user-images.githubusercontent.com/16045377/152141858-314ce500-6454-4958-b59e-0ef33e71d1a8.png)

## Types of changes


## Checklist:
